cpp/src/client/printMessage.cc

```diff
@@ -4,6 +4,10 @@
 void printMessage(const std::string &msg) {
   cout << "\r";
   auto tokens = split(msg, DELIM, 3);
+  if (tokens.size() != 3) {
```

Comment:
why this?

Reply:
If token.size() != 3 it means received msg is the part of previously received msg, so need of formatting simply print the msg.

Reply:
what if someone sends a normal msg (that needs to be formatted) while the larger message is in print/transition?

Reply:
add a comment here why it is necessary